src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write your code here
+const logo = document.getElementsByClassName('logo')[0];
```

Comment:
Ensure that there is an element with the class 'logo' in the HTML. If not, `logo` will be `undefined`, and the event listener will not be attached.